src/components/Settings/People/People.component.js

```diff
@@ -95,126 +90,125 @@ const People = ({
         disableSubmit={!isLogged}
       >
         <Paper>
-          <List>
-            <ListItem>
-              <div className="Settings__UserIcon__Container">
-                <UserIcon />
-              </div>
-              <ListItemText primary={name} />
-              <ListItemSecondaryAction className="Settings--secondaryAction">
+          <Box px={3} py={4}>
+            {/* Icono y Logout */}
+            <Box
+              display="flex"
+              flexDirection="column"
+              alignItems="center"
+              mb={4}
+            >
+              <UserIcon />
+              <Typography variant="h6" sx={{ mt: 2, textAlign: 'center' }}>
```

Comment:
The `sx` prop is not officially supported in v4 of material UI. And I think here this prop is not making any change